### ISSUE TITLE
How to disable hardware acceleration?

### ISSUE THREAD

**Issue by Evafan1337** [minbrowser/min#673]
Hello!
I use Debian 9 from VMware, and sometimes i have some green screens and black boxes while serfing the Net. My GPU in Linux is virtual and it is of course is very weak.
Hardware is ok (4 core cpu, 4gb ram, on my main OS ( Windows  10) ).
The question is: how can i disable hardware acceleration? Because i didn't find it in browser setting.
I am so sorry for silly quiestions, maybe this questions were asked before, i will be glad to read this topics.

**Comment by PalmerAL** [comment]
You're right, there isn't an option for it in the UI. You should be able to disable hardware acceleration temporarily by launching Min with the `--disable-gpu` flag, although that probably isn't a permanent solution.
